src/httpserver.cpp

```diff
@@ -106,19 +105,15 @@ class WorkQueue
      */
     ~WorkQueue()
```

Comment:
On second thought, I'm going to keep it, the precondition comment is still relevant.